.github/workflows/andi.yml

```diff
@@ -0,0 +1,64 @@
+name: Andi
+on:
+  push:
+    branches:
+      - master
+    paths:
+    - 'apps/andi/**'
```

Comment:
pros: allows us to truly skip steps (not spin up a vm for 45 seconds to check if it changed)
cons: lots of copy and paste for library based apps we want to check

Reply:
Adding new lines here isn't too horrible a cost to pay in my opinion